seedlet/assets/css/ie.css

```diff
@@ -6535,6 +6535,26 @@ button[data-load-more-btn],
 	padding-right: 20px !important;
 }
 
+.wp-block-jetpack-layout-grid .wp-block-jetpack-layout-grid-column > * {
+	margin-top: 19.98px;
```

Comment:
What's up with these partial pixel measurements? Is that mirroring something from the Grid block?

Reply:
Ah there is a postCSS plugin to transform `calc()` to static values wherever possible for IE. This is coming from a `calc( 0.666 * var(--global--spacing-horizontal) );`.